feature(#126): TeaBrewingGuide 컴포넌트 구현

suubiin opened the pull request:
### feature(#126): TeaBrewingGuide 컴포넌트 구현

### PR 유형

- [x] 새로운 기능 추가

### PR 체크 리스트

<!-- PR이 다음 요구 사항을 충족하는지 확인하세요. -->

- [x] 커밋 메시지 컨벤션에 맞게 작성했습니다.
- [x] 변경 사항에 대한 테스트를 했습니다.(버그 수정/기능에 대한 테스트)

### 🔎 작업 내용

<!-- 어떤 작업을 하셨는지 상세하게 작성해주세요. -->

- TeaBrewingGuide 컴포넌트 구현
- 티 양, 물 양, 온도, 시간, 각각의 단위 변수 처리
- 스토리북 생성
  <br />

### 🔧 앞으로의 과제

- 내일 할 일을 적어주세요.
  <br />

### 이슈

<!-- 이슈 키워드와 함께 #을 입력한 후 이슈 번호를 선택해주세요. -->
<!-- 예시 : resolves #1 -->

resolves #126

suubiin committed:
feature: TeaBrewingGuide 컴포넌트 구현
- TeaBrewingGuide 컴포넌트 구현
- 티 양, 물 양, 온도, 시간, 각각의 단위 변수 처리
- 스토리북 생성

woolen-h commented on src/components/TeaCard/TeaBrewingGuide.tsx:
```diff
@@ -0,0 +1,89 @@
+               <span className={unitClass}>{timeUnit}</span>
+            </dd>
+         </dl>
+      </section>
```
```tsx
function BrewingDetail({
   iconClassName,
   value,
   unit,
   srLabel,
}: {
   iconClassName: string;
   value: number;
   unit: string;
   srLabel: string;
}) {
   return (
      <dl className={containerClass}>
         <dt>
            <span className={`${iconClassName} ${iconClass}`} aria-hidden="true"></span>
            <span className="sr-only">{srLabel}</span>
         </dt>
         <dd className={containerClass}>
            <p className={valueClass}>{value}</p>
            <span className={unitClass}>{unit}</span>
         </dd>
      </dl>
   );
}
```
가독성을 위해 요런 식으로 반복되는 구조는 서브컴포넌트 파일 내에 선언해서 써주면 어떨까 싶습니다!

happyhye reviewed (approved):
검토 완료 👍
한울님 코멘트 반영해주시면 될 것 같아요~!

suubiin committed:
Merge branch 'develop' into feature/iconrecipe-choi

suubiin committed:
refactor: TeaBrewingGuide 피드백 반영 수정
- 가독성을 위해 BrewingDetail 서브컴포넌트로 분리
- tailwind.config.js: box-shadow 명칭 변경
- 수정사항 반영 스토리북 코드 수정

woolen-h reviewed (approved):
확인 완료했습니다~~~ 짱짱!
